### PR TITLE
Create header

### PR DESCRIPTION
**Pull request by nikitasakau**
Not sure that the correct implementation has done. Wait for comments :)

### CODE REVIEW

**Comment by nikitasakov** [comment]
[I as user want to see header with app logo at every page](https://trello.com/c/JK75EEKs/8-i-as-user-want-to-see-header-with-app-logo-at-every-page)

File: client/src/Header.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.Header {
```
Comment (AleksSenkou) [inline client/src/Header.css]: thoughts about having css styles downcased? `.header`

@datarockets/cuties ?

**Review by AleksSenkou — APPROVED** [review]
LGTM with suggestion

**Comment by DDKatch** [comment]
LGTM

**Pull request merged**